internal/pkg/diagnostics/diagnostics.go

```diff
@@ -282,12 +312,14 @@ func zipLogs(zw *zip.Writer, ts time.Time) error {
 		return err
 	}
 
-	if err := collectServiceComponentsLogs(zw); err != nil {
-		return fmt.Errorf("failed to collect endpoint-security logs: %w", err)
+	if collectServices {
+		if err := collectServiceComponentsLogs(zw); err != nil {
+			return fmt.Errorf("failed to collect endpoint-security logs: %w", err)
+		}
 	}
 
 	// using Data() + "/logs", for some reason default paths/Logs() is the home dir...
-	logPath := filepath.Join(paths.Home(), "logs") + string(filepath.Separator)
+	logPath := filepath.Join(pathsHome, "logs") + string(filepath.Separator)
```

Comment:
Not introduced in this change but I am just curious: why do we need an extra separator ad the end of logPath ?

Reply:
dunno, probably something with a zip writer